smartmodule/regex-filter/Makefile

```diff
@@ -1,5 +1,11 @@
 default: build
 
-build:
-	rustup target add wasm32-unknown-unknown
-	cargo build --profile release-lto --target wasm32-unknown-unknown
+setup:
```

Comment:
not show stopper but should reuse common targets and constants so we have consistent and repeatable build

Reply:
yes, here is our reminder issue https://github.com/infinyon/fluvio/issues/3998